dags/benchmark/pytorchxla_torchbench.py

```diff
@@ -31,9 +31,9 @@
 SCHEDULED_TIME = "0 17 * * *" if composer_env.is_prod_env() else None
 
 with models.DAG(
-    dag_id="pytorchxla-torchbench",
-    schedule=SCHEDULED_TIME,
-    tags=["pytorchxla", "nightly", "torchbench"],
+    dag_id="pytorch_nightly_torchbench",
+    schedule=None,
```

Comment:
Is this intentional? Do we want to disable this test's schedule?

Reply:
Oh.. I think this is introduced due to the merge a while ago. The head has been changed already, I will update in the next commit.